app/services/giphy_service.rb

```diff
@@ -0,0 +1,12 @@
+class GiphyService
+  def giphy_img?(source)
```

Comment:
This could be in `app/services/giphy/image.rb` and a class method thoo, since it's not using any instance data

Reply:
Sure, that makes a lot more sense. I'll get to it soon as I finish a couple of course videos I'm watching 😄.